src/FrameClient.js

```diff
@@ -416,6 +416,7 @@ class FrameClient {
       "PublicRep",
       "PublishContentVersion",
       "QParts",
+      "RecordWriteToken",
```

Comment:
I think this may open up a security hole. A hostile app could submit an arbitrary node URL and cause outer frame elv-client-js instance to send all http requests for the write token to a third-party (even those requested by other apps hosted in the same elv-core-js page).

Reply:
Actual impact may be mitigated by need to know the write token, but still seems sketchy